zebra-scan/Cargo.toml

```diff
@@ -77,6 +85,11 @@ zcash_note_encryption = { version = "0.4.0", optional = true }
 
 zebra-test = { path = "../zebra-test", version = "1.0.0-beta.37", optional = true }
 
+# zebra-scanner binary dependencies
+tracing-subscriber = "0.3.18"
```

Comment:
```suggestion
tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
```

Reply:
It is just enough to add the feature or do we need to use it in the code ?

Reply:
> It is just enough to add the feature or do we need to use it in the code ?

I think it's already being used in the code [here](https://github.com/ZcashFoundation/zebra/pull/8608/files#diff-c0325cc03c0149c3a95151814a89b49d3c0cb7de88b1a4c31e0ae85865e89663R38), it was failing to compile without it locally.

Reply:
Sorry, you are right, it seems i did had the change already locally. Re added it in https://github.com/ZcashFoundation/zebra/pull/8608/commits/1f056c6c45c1aad5d20645049b2d1fb56976675d